core/SConscript.bootloader

```diff
@@ -228,6 +246,12 @@ def cargo_build():
     features.append("bootloader")
     features.extend(FEATURES_AVAILABLE)
 
+    if NEW_RENDERING:
+        features.append('new_rendering')
+
+    if TREZOR_MODEL in ('T',):
```

Comment:
more models need antialiasing

Reply:
fixed d6bdaa93326ebaf03249a9899595b5ad04323f14

Reply:
`DISC1 `and `DISC2` too

Reply:
I've moved `ui_antialiasing` from build scripts to `cargo.toml`, see f55c42b94037fbd1658fbb4f28c577b298b409df

Reply:
remained in `Sconscript.bootloader`

Reply:
i hope, it's final fix, see a4fd40fb1de7a41ea9affdcb0b4f3ff84b5dbb9e